Sources/Brave/Frontend/Browser/BrowserViewController/OpenSearch/BrowserViewController+OpenSearch.swift

```diff
@@ -183,7 +183,27 @@ extension BrowserViewController {
   }
 
   private func addSearchEngine(_ engine: OpenSearchEngine) {
-    let alert = ThirdPartySearchAlerts.addThirdPartySearchEngine(engine) { alertAction in
+    var customEngineAlert: UIAlertController
+
+    if let existingEngine = profile.searchEngines.orderedEngines.first(where: { $0.shortName == engine.shortName }) {
```

Comment:
Use `caseInsensitiveCompare` instead of `==` to ensure you ignore case

Reply:
Fix at https://github.com/brave/brave-ios/pull/7721/commits/02941ed05e557a8831d25723423aba8263c62d2b